assets/js/scripts.js

```diff
@@ -7,3 +7,20 @@ menuTrigger.onclick = function() {
   menuTrigger.classList.toggle('is-active');
   body.classList.toggle('lock-scroll');
 };
+
+function openTab(evt, tabName) {
+  let i;
+  const tabContent = document.getElementsByClassName('tab-content');
+  for (i = 0; i < tabContent.length; i++) {
```

Comment:
Use modern for-loops:

```suggestion
for (const tabContent of tabContents) {
```

Reply:
Updated.